tests/integration/test_remoted/test_active_response/test_active_response_send_ar.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import pytest
+import time
+
+import wazuh_testing.remote as remote
+import wazuh_testing.tools.agent_simulator as ag
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.sockets import send_active_response_message
+from wazuh_testing import UDP, TCP, TCP_UDP
+
+# Marks
+pytestmark = pytest.mark.tier(level=1)
+
+# Configuration
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_test_active_response.yaml')
+
+parameters = [
+    {'PROTOCOL': TCP, 'PORT': '1514'},
+    {'PROTOCOL': UDP, 'PORT': '1514'},
+    {'PROTOCOL': TCP_UDP, 'PORT': '1514'}
+
+]
+metadata = [
+    {'protocol': TCP, 'port': '1514'},
+    {'protocol': UDP, 'port': '1514'},
+    {'protocol': TCP_UDP, 'port': '1514'}
+]
```

Comment:
Add here more tests changing the port a different one too

Reply:
Done in 3cc7447